Sources/App/Authentication/UserAuthenticator.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UserAuthenticator.swift
+//  
+//
+//  Created by Rob Maltese on 8/17/22.
+//
+
+import Vapor
+import Fluent
+
+extension User: ModelAuthenticatable {
+    static let usernameKey = \User.$email
```

Comment:
username is email? 😄

Reply:
Ahhh!

Reply:
Yes. Unfortunately it seems it must stay that way because of the way Vapor has the protocol setup. Are we able to edit the Vapor protocol? When I changed it, I got yelled at saying I was missing protocol stubs.

Reply:
Sounds good. Leave a comment!